Sources/XMLCoder/Decoder/XMLKeyedDecodingContainer.swift

```diff
@@ -246,9 +253,10 @@ extension XMLKeyedDecodingContainer {
             )
         }
 
+        let inlined = key.stringValue == ""
```

Comment:
Instead of reimplementing `inlined`/`isInlined`, can we make `isInlined` a `internal` computed var on `CodingKey`?